python_scripts/03_categorical_pipeline_column_transformer.py

```diff
@@ -55,9 +55,10 @@
 # ```{caution}
 # Here, we know that `object` data type is used to represent strings and thus
 # categorical features. Be aware that this is not always the case. Sometimes
-# `object` data type could contain other type of information (e.g. dates that
-# were not parsed) and you should manually introspect the content of your
-# dataframe to not wrongly use `make_column_selector`.
+# `object` data type could contain other type of information, such as dates that
+# were not parsed and can then relate to age (a quantity). In a more general
```

Comment:
A couple of thoughts about this change. I don't think that this is enough to remove the ambiguity in the quiz. I would be inclined to get a small paragraph to add more information.

The quiz refers to a "year" and not a date indeed. The year can already be encoded as a numerical value and thus a quantity.

A date that is not parsed properly will be a string, with a format such as "dd-mm-yyyy". It is indeed what we are referring to in the text here. So this is not a quantity. If you use an ordinal encoder, you will transform it to a quantity but it will most probably go sideways. Indeed, the ordering might not be respected and then the encoded value will not be linked to age.

So in the current paragraph, we could make things more explicit regarding the date format (what I try to explain in the previous paragraph). In addition, we can add a new note regarding quantity as year/day/month and why sometimes some numerical values could be considered either a category or a numerical value.